### PR TITLE
tdm: init at 2.07

### PR DESCRIPTION
**Pull request by cizra**
###### Motivation for this change
Add The Dark Mod http://www.thedarkmod.com/main/ — a FPS game inspired by the Thief series

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Runs.

### CODE REVIEW

**Comment by Lassulus** [comment]
test it with nix-review, this is the best thing since baked bread. 
Is the license really gpl3 though? I can't find anything about it on the webpage.

**Comment by cizra** [comment]
@Lassulus 

The Dark Mod was originally released as a total conversion modification for Doom 3, but with the release of version 2.0 it became completely standalone on the id Tech 4 game engine. The game's source code is licensed under the GNU General Public License 3 and all other non-software components are under Creative Commons BY-NC-SA.[3]

https://en.wikipedia.org/wiki/The_Dark_Mod
https://svn.thedarkmod.com/publicsvn/darkmod_src/trunk/LICENSE.txt

File: pkgs/games/tdm/default.nix [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  '';
+
+  enableParallelBuilding = true;
+  sconsFlags = [ "BUILD=release" "TARGET_ARCH=x64" ];
+  NIX_CFLAGS_COMPILE = ["-Wno-error=format-security"];
+  meta = with stdenv.lib; {
+    description = "The Dark Mod - stealth FPS inspired by the Thief series";
+    homepage = http://www.thedarkmod.com;
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ elmo ];
```
Comment (Lassulus) [inline pkgs/games/tdm/default.nix]:
```suggestion
    maintainers = with maintainers; [ cf6b88f ];
```

File: pkgs/games/tdm/default.nix [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  postInstall = ''
+    wrapProgram $out/bin/tdm --suffix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libGLU_combined ]}
+  '';
+
+  enableParallelBuilding = true;
+  sconsFlags = [ "BUILD=release" "TARGET_ARCH=x64" ];
+  NIX_CFLAGS_COMPILE = ["-Wno-error=format-security"];
+  meta = with stdenv.lib; {
+    description = "The Dark Mod - stealth FPS inspired by the Thief series";
+    homepage = http://www.thedarkmod.com;
```
Comment (Lassulus) [inline pkgs/games/tdm/default.nix]:
```suggestion
    homepage = "http://www.thedarkmod.com";
```

Comment (Lassulus) [reply pkgs/games/tdm/default.nix]: https://github.com/NixOS/rfcs/pull/45

**Comment by Lassulus** [comment]
@GrahamcOfBorg build tdm

**Pull request merged**